src/components/ble/MusicService.h

```diff
@@ -31,40 +34,40 @@ namespace Pinetime {
     class SystemTask;
   }
   namespace Controllers {
-    
+
```

Comment:
Seems like a potential autoformatter fight, I wouldn't include this in this PR

Reply:
At some point cmake-format should be configured and applied, and that'll act as an authoritative source.

Reply:
```suggestion
    
```